servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -2140,7 +2144,7 @@ void RendererSceneRenderRD::render_buffers_configure(RID p_render_buffers, RID p
 		if (rb->view_count > 1) {
 			tf.texture_type = RD::TEXTURE_TYPE_2D_ARRAY;
 		}
-		tf.format = RD::DATA_FORMAT_R16G16B16A16_SFLOAT;
+		tf.format = render_buffers_get_color_format();
```

Comment:
probably would use  `_render_buffers_get_color_format()` since this should be virtual protected.

Reply:
Sounds good